proto/poktroll/migration/params.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package poktroll.migration;
+
+import "amino/amino.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/pokt-network/poktroll/x/migration/types";
+option (gogoproto.stable_marshaler_all) = true;
+
+// Params defines the parameters for the module.
+message Params {
+  option (amino.name) = "poktroll/x/migration/Params";
+  option (gogoproto.equal) = true;
+
```

Comment:
Lots of empty lines?

Reply:
Happy to make the change. This was generated and I just didn't edit it.